__tests__/models/application.test.ts

```diff
@@ -0,0 +1,32 @@
+import { Application } from '@models';
```

Comment:
@ketanreliefapps it applies for this model and all models where we would find something like:

``` ts// handle cascading deletion for applications
addOnBeforeDeleteMany(applicationSchema, async (applications) => {
  const pages = applications.reduce((acc, app) => acc.concat(app.pages), []);
  // Delete pages, roles and channels
  await Page.deleteMany({ _id: { $in: pages } });
  await Role.deleteMany({ application: { $in: applications } });
  await Channel.deleteMany({ application: { $in: applications } });
});
```

we should also test if, when adding new linked objects to this object, and we delete the object, that the linked objects are deleted as they should